Explorer/Assets/DCL/Multiplayer/Connections/Systems/DebugRoomsSystem.cs

```diff
@@ -39,13 +40,14 @@ public DebugRoomsSystem(
             IDebugContainerBuilder debugBuilder,
             IObjectPool<DebugRoomIndicatorView> roomIndicatorPool) : base(world)
         {
+            enabled = true;
```

Comment:
why are you setting it to true here if a few lines after it's getting set to `infoWidget != null`? 🤔

Reply:
removed at https://github.com/decentraland/unity-explorer/pull/4689/commits/960cda95fbbe6893e0a2314c0097acc5824c7a2e